src/components/button/Button.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { handleUrl } from '../../utilities'
+import { StyledButton } from './Button.styles'
+import {
+  ButtonAsAnchorProps,
+  ButtonProps,
+  ButtonSignature,
+} from './Button.types'
+
+/**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing JSDoc for parameter 'props'.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)